lib/master.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file In order to load huge volumes of polygon data into memory without
+ *    breaking Node (due to its low heap-size limits), the package forks a
+ *    worker process per polygon layer/shapefile. This module contains
+ *    functions for initializing them and searching them.
+ */
+
+'use strict';
+
+var childProcess = require( 'child_process' );
+var peliasConfig = require( 'pelias-config' );
+var path = require( 'path' );
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'admin-lookup:master' );
+
+/**
+ * For every desired administrative layer, specifies its name, the partial path
+ * of the corresponding Quattroshapes file, and the properties to extract from
+ * it.
+ */
+var quattroAdminLevels = [
```

Comment:
This might also be better off in the config file, with defaults set here perhaps. Would also be very helpful to provide an example config.json file (./etc/config.json) with all the expected settings. Not having a local config.json setup in the default path made it harder for me to run the tests out of the box.